src/malloy/data/bigquery/bq_connection.py

```diff
@@ -104,29 +104,35 @@ def _to_struct_def(self, table, schema):
 
   def _to_array_struct_def(self, name):
     return {
-      "type": "struct",
-      "name": name,
-      "dialect": "standardsql",
-      "structSource": {"type": "nested"},
-      "structRelationship": {
-        "type": "nested",
-        "field": name,
-        "isArray": True,
-      },
+        "type": "struct",
+        "name": name,
+        "dialect": "standardsql",
+        "structSource": {
+            "type": "nested"
+        },
+        "structRelationship": {
+            "type": "nested",
+            "field": name,
+            "isArray": True,
+        },
```

Comment:
Nothing actually changed in this file, I just figured out how to turn on the formatter...